src/components/AssetInfo/TokenSideBar.test.tsx

```diff
@@ -7,12 +7,147 @@ describe("tokenSideBar", () => {
     const wrapper = mount(
       <TokenSideBar
         isSideBarExpand={true}
-        holderAddress="0xA"
-        beneficiaryAddress="0xB"
+        holderAddress=""
+        beneficiaryAddress=""
         approvedBeneficiaryAddress=""
         handler={() => {}}
       />
     );
     expect(wrapper.find("h2").text()).toStrictEqual("Manage Asset");
   });
+
+  // holder === bene, approved bene not exists
+  it("should show holder and beneficiary role, holder view, only 2 correct fields", () => {
+    const wrapper = mount(
+      <TokenSideBar
+        isSideBarExpand={true}
+        holderAddress="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
+        beneficiaryAddress="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
+        approvedBeneficiaryAddress=""
+        handler={() => {}}
+      />
+    );
+
+    expect(wrapper.find("TokenSideBarRole h4").text()).toStrictEqual("Holder and Beneficiary");
+    expect(wrapper.find("TokenSideBarHolder")).toHaveLength(1);
+    expect(wrapper.find("TokenSideBarHolder section")).toHaveLength(2);
+    wrapper.find("TokenSideBarHolder section").forEach((node, index) => {
+      switch (true) {
+        case index === 0:
+          expect(node.find("h4").text()).toStrictEqual("Transfer Ownership");
+          break;
+        case index === 2:
+          expect(node.find("h4").text()).toStrictEqual("Surrender Document");
+          break;
+        default:
+      }
```

Comment:
can give the ownership and surrender and other sections id and use that. later on if you change the order test will fail.